src/i18n/en/common.json

```diff
@@ -102,6 +102,34 @@
       }
     }
   },
+  "favouritesPage": {
+    "title": "You have added following plots to favourites",
+    "targetCard": {
+      "removeButton": "Remove from list",
+      "table": {
+        "detailedPlanIdentifier": "Detailed plan identifier",
+        "detailedPlanState": "Detailed plan state",
+        "financingMethod": "Financing method",
+        "firstSuitableConstructionYear": "First suitable construction year",
+        "hitas": "HITAS",
+        "infoLinks": "Details",
+        "intendedUse": "Intended use",
+        "management": "Management method",
+        "address": "Address",
```

Comment:
I think you're extracting these strings manually since they're not in the alphabetical order; you can and should just run `yarn run i18n` to collect them automatically instead.

Reply:
...TIL

Reply:
I mean, I put it in the readme and you reviewed that commit, so I assumed you were aware of it :D